content/en/docs/tasks/administer-cluster/controller-manager-leader-migration.md

```diff
@@ -21,15 +21,15 @@ Leader Migration provides a mechanism in which HA clusters can safely migrate "c
 
 Leader Migration can be enabled by setting `--enable-leader-migration` on `kube-controller-manager` or `cloud-controller-manager`. Leader Migration only applies during the upgrade and can be safely disabled or left enabled after the upgrade is complete.
 
-This guide walks you through the manual process of upgrading the control plane from `kube-controller-manager` with built-in cloud provider to running both `kube-controller-manager` and `cloud-controller-manager`. If you use a tool to administrator the cluster, please refer to the documentation of the tool and the cloud provider for more details.
+This guide walks you through the manual process of upgrading the control plane from `kube-controller-manager` with built-in cloud provider to running both `kube-controller-manager` and `cloud-controller-manager`. If you use a tool to deploy and administrator the cluster, please refer to the documentation of the tool and the cloud provider for specific instructions of the migration.
```

Comment:
```suggestion
This guide walks you through the manual process of upgrading the control plane from `kube-controller-manager` with built-in cloud provider to running both `kube-controller-manager` and `cloud-controller-manager`. If you use a tool to deploy and manage the cluster, please refer to the documentation of the tool and the cloud provider for specific instructions of the migration.
```

Reply:
You could use "administer" if you don't like the word "manage".

Reply:
Thanks for the suggestion to add "deploy".